CMake/FollyConfigChecks.cmake

```diff
@@ -85,12 +85,12 @@ string(REGEX REPLACE
 
 check_symbol_exists(pthread_atfork pthread.h FOLLY_HAVE_PTHREAD_ATFORK)
 
-check_symbol_exists(accept4 sys/socket.h FOLLY_HAVE_ACCEPT4)
+check_symbol_exists(accept4 "sys/socket.h;linux/socket.h" FOLLY_HAVE_ACCEPT4)
 check_symbol_exists(getrandom sys/random.h FOLLY_HAVE_GETRANDOM)
 check_symbol_exists(preadv sys/uio.h FOLLY_HAVE_PREADV)
-check_symbol_exists(pwritev sys/uio.h FOLLY_HAVE_PWRITEV)
```

Comment:
Was this a typo?

Reply:
Thanks for looking at that existing patch version.   As suggested above, the right solution is probably to have a new macro that detects whether linux-libc-dev is version 4.x or 5.x, and to use it to fix the clock_gettime, pthread_atfork, pipe2 and accept4 all with one test.    I have not had time yet to analyze whether all 4 system calls changed between libc-header versions.  What puzzles me is that the boost part of the build appears to run its own separate detection step, so that any fix would have to be made in two cmake scripts.   Is that correct?